src/components/recaptcha.js

```diff
@@ -0,0 +1,91 @@
+(() => ({
+  name: 'Recaptcha',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: [],
+  orientation: 'HORIZONTAL',
+  jsx: (
+    <div className={classes.root}>
+      {(() => {
+        const isDev = B.env === 'dev';
+        const isReady = () =>
+          typeof window !== 'undefined' &&
+          typeof window.grecaptcha !== 'undefined' &&
+          typeof window.grecaptcha.ready;
+
+        const [validated, setValidated] = useState(false);
+        const [ready, setReady] = useState(isReady());
+        const [widget, setWidget] = useState(null);
+        const element = useRef(null);
+        let readyCheck;
+
+        const updateReadyState = () => {
+          if (isReady()) {
+            setReady(true);
+            clearInterval(readyCheck);
+          }
+        };
+
+        useEffect(() => {
+          if (ready && element.current && !widget && options.sitekey) {
+            setWidget(
+              window.grecaptcha.render(element.current, {
+                sitekey: options.sitekey,
+                callback: () => {
+                  setValidated(true);
+                },
+              }),
+            );
+          }
+        }, [ready, element.current]);
+
+        useEffect(() => {
+          if (!ready) {
+            readyCheck = setInterval(updateReadyState, 1000);
+          }
+        }, []);
+
+        useEffect(() => {
+          const script = document.createElement('script');
+          script.src =
+            'https://www.google.com/recaptcha/api.js?render=explicit';
+          script.async = true;
+          script.defer = true;
+
+          document.body.appendChild(script);
+
+          return () => {
+            document.body.removeChild(script);
+          };
+        }, []);
```

Comment:
This is a very hackie way of using a script for this component and not something that we want to support.

Reply:
Do you have a recommended alternative way to include a script tag in a component?

Reply:
No I believe this is currently the only solution for using a script in a component. 

It is also possible to add this to a bundle but we don't want to add this as another dependency to our default bundle. 

We will have to figure out a proper solution to be able to support this.